website/core/Footer.js

```diff
@@ -35,14 +35,14 @@ class Footer extends React.Component {
           </a>
           <div>
             <h5>Docs</h5>
-            <a href={this.docUrl('doc1.html', this.props.language)}>
-              Getting Started (or other categories)
+            <a href={this.docUrl('#', this.props.language)}>
```

Comment:
I think these should still link to some page, we just need to fill-in that page.

Instead of calling it `doc1.html` it would be `getting-started.html` or something.